src/test/scala/com/bloomberg/sparkflow/serialization/HashingTest.scala

```diff
@@ -58,4 +61,15 @@ class HashingTest extends FunSuite {
 
     assert(allSignatures.size == 4)
   }
+
+  test("caseHashing"){
+    setLogLevel("DEBUG")
+    println(s"fieldObjects: ${getFieldObjects(HashingSample.result)}")
+    println(s"result: ${HashingSample.result.getSignature}")
```

Comment:
probably don't print this much, put a simple assert here, just enough that it's able to calculate the signature is enough for now